client/src/components/Footer/index.tsx

```diff
@@ -19,7 +19,6 @@ function Footer(): JSX.Element {
               <Link className='inline' to='/donate'>
                 {t('footer.donate-link')}
               </Link>
```

Comment:
We should probably have this and `footer.donate-text` within a `Trans` component to allow the sentence to be one string.

Reply:
Yes, that will make translation work a lot easier. I will make that change too.

Reply:
@ShaunSHamilton Done. Could you please review again? Thank you.

Reply:
`Trans` components are a pain to work with, unfortunately, since it's not at all intuitive what the translation string (in `translation.json`) should be. It's worth taking a look at their [docs](https://react.i18next.com/latest/trans-component), though. They're fairly comprehensive.

Here's a case where we use them:
https://github.com/freeCodeCamp/freeCodeCamp/blob/ae7535299ea98189acdb3c75ce1d4f9e43975e80/client/src/templates/Challenges/components/help-modal.tsx#L53-L62
and here's the corresponding key:
https://github.com/freeCodeCamp/freeCodeCamp/blob/ae7535299ea98189acdb3c75ce1d4f9e43975e80/client/i18n/locales/english/translations.json#L275

Let me know if that helps or you need more context and good luck!

Reply:
Hi @ojeytonwilliams I made changes to use `Trans` in the next commit. It seems working fine to me. Would you please point out if I'm missing something? Thank you.

Reply:
Sorry, @sidemt, I saw that the component was missing from the snapshot and assumed that it was missing from the page. It's not, and I should have checked that!

So, the only things we need to fix are the tests, since they should include the translated text in the snapshot.

That said, we should probably be consistent about placeholders.  Clearly, as you've shown, we don't need them, but I wonder if they're helpful. For instance

```jsx
<Trans i18nKey='footer.donate-text'>
  <Link className='inline' to='/donate' />
</Trans>
```
 vs 
```jsx
<Trans i18nKey='footer.donate-text'>
  placeholder one
  <Link className='inline' to='/donate'>
    placeholder two
  </Link>
</Trans>
```

It's a touch more verbose, but it makes it obvious what the final component will be once the translation is done.  What do you think @sidemt?

Reply:
The placeholders would be the English text? Yeah it makes it easier looking at the code

Reply:
The code would look like the second one.  We'd need to adjust the keys (at a guess it would be `You can <1>make a tax-deductible donation here</1>.`).  That's it.